cloud-spanner-spring-data-r2dbc/src/main/java/com/google/cloud/spanner/r2dbc/springdata/SpannerR2dbcDialect.java

```diff
@@ -87,4 +93,15 @@ public LimitClause limit() {
   public LockClause lock() {
     return LOCK_CLAUSE;
   }
+
+  @Override
+  public Collection<? extends Class<?>> getSimpleTypes() {
+    return Arrays.asList(JsonWrapper.class);
+  }
+
+  @Override
+  public Collection<Object> getConverters() {
+    return Arrays.asList(new JsonToMapConverter(this.gson), new MapToJsonConverter(this.gson));
```

Comment:
```suggestion
    return Arrays.asList(new JsonToMapConverter<>(this.gson), new MapToJsonConverter<>(this.gson));
```

This will explicitly set parameterized types to `<Object, Object>`.